### PR TITLE
Replace non-ascii string under test/

### PR DESCRIPTION
**Pull request by oomichi**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

These string seems written unintentionally.
This replaces them.

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: test/integration/benchmark/jsonify/main.go [hunk 1]
```diff
@@ -72,7 +72,7 @@ func run() error {
 	data := PerfData{Version: "v1"}
 	for _, benchMarks := range benchmarkSet {
 		for _, benchMark := range benchMarks {
-			data.DataItems = appendIfMeasured(data.DataItems, benchMark, benchparse.NsPerOp, "time", "μs", benchMark.NsPerOp/1000.0)
+			data.DataItems = appendIfMeasured(data.DataItems, benchMark, benchparse.NsPerOp, "time", "us", benchMark.NsPerOp/1000.0)
```
Comment (SataQiu) [inline test/integration/benchmark/jsonify/main.go:75]: Thanks @oomichi 
Do we really need to replace it?  `μs` is easier to recognize as a unit of time.

Comment (oomichi) [reply test/integration/benchmark/jsonify/main.go:75]: Ah, when I opened this file with `vim` that seems a Chinese character and I could not see the meaning.
I am fine to keep this char as we see it on the browser.

Comment (oomichi) [reply test/integration/benchmark/jsonify/main.go]: @SataQiu Is current updated PR good for you?

**Comment by atoato88** [comment]
/lgtm
You mean that this PR replaces `”` : `%E2%80%9C`(UTF-8 hex) with `"` : `%22`.

**Pull request merged**